encrypted-config-value-module/src/main/java/com/palantir/config/crypto/jackson/EncryptedConfigMapperUtils.java

```diff
@@ -37,4 +37,12 @@ private EncryptedConfigMapperUtils() {}
         return mapper.treeToValue(substitutedNode, clazz);
     }
 
+    public static <T> T getConfig(String configFileContent, Class<T> clazz, ObjectMapper mapper)
+            throws JsonParseException, JsonMappingException, IOException {
+        JsonNode configNode = mapper.readValue(configFileContent, JsonNode.class);
+        JsonNode substitutedNode = JsonNodeVisitors.dispatch(
+                mapper.valueToTree(configNode),
+                new JsonNodeStringReplacer(new DecryptingVariableSubstitutor()));
```

Comment:
If we're adding this to reduce allocations from continuous config reloading, why not re-use a single instance of this rather than create a new one each time? is it stateful?

Reply:
Does not look like it